ct-web-lit/src/components/CustomizableTagName.ts

```diff
@@ -0,0 +1,13 @@
+import { LitElement, html } from 'lit';
+
+export class CustomizableTagName extends LitElement {
+    private static tagName = `customizable-tag-name-component`;
```

Comment:
```suggestion
    static #tagName = `customizable-tag-name-component`;
```

Feel free to skip, but would be nice if native features are used, the `tsconfig.json` probably needs to be adjusted for this. `tsconfig.json` in `ct-web` could be reuse if you want to adjust it:

https://github.com/sand4rt/playwright-ct-web/blob/2d6360947e8f23675028570a6752d89f9d0f6d2f/ct-web/tsconfig.json#L1-L24